R/db-odbc-mssql.R

```diff
@@ -47,6 +47,7 @@
       atan2         = sql_prefix("ATN2"),
       ceil          = sql_prefix("CEILING"),
       ceiling       = sql_prefix("CEILING"),
+      sd            = sql_prefix("STDEV"),
```

Comment:
I was thinking scalar could be used for cases when you want to run an operation against the result of the entire dataset, like this: 

```
> tbl(con, "iris") %>%
+   mutate(A_sd = sd(Sepal.Length),
+          s = Sepal.Length / sd(Sepal.Length)) %>%
+   select(Sepal.Length,
+          A_sd,
+          s)
# Source:   lazy query [?? x 3]
# Database: Microsoft SQL Server
#   12.00.4422[rstudioadmin@EC2AMAZ-O5QKKM8/airontime]
   Sepal.Length      A_sd        s
          <dbl>     <dbl>    <dbl>
 1          5.1 0.8280661 6.158928
 2          4.9 0.8280661 5.917402
 3          4.7 0.8280661 5.675875
```